hal_st/stm32fxxx/DmaStm.cpp

```diff
@@ -213,10 +282,21 @@ namespace hal
 #endif
 #if defined(DMA_FLAG_TC8)
             DMA_FLAG_TC8,
+#endif
 #endif
         };
         const std::array streamToHTIF
         {
+#if defined(STM32WBA)
```

Comment:
```suggestion
#ifdef DMA_CFCR_HTF
```

Reply:
etc, please check the rest of the file as well.

Reply:
Good, fixed.